python-avd/pyproject.toml

```
@@ -25,6 +25,7 @@ dependencies = [
     "deepmerge>=1.1.0",
     "Jinja2>=3.0",
     "requests>=2.27.0",
+    "passlib==1.7.4",
```

please don't pin dependencies
```suggestion
    "passlib>=1.7.4",
```